Code/GraphMol/FindStereo.cpp

```diff
@@ -943,17 +982,19 @@ std::vector<StereoInfo> runCleanup(ROMol &mol, bool flagPossible,
   boost::dynamic_bitset<> bondsInPlay(mol.getNumBonds());
   bondsInPlay.set();
 
+#define LOCAL_CANON 0
+#if LOCAL_CANON
```

Comment:
There were a couple of variables that were allocated that were only used in the LOCAL-CANON part.   I encountered some double-delete or valgrind error, so I just tried to isolate those variables to only be included if LOCAL-CANON was in affect. Really just cleanup so that it is not allocating and deleting variables it is not using.